README.md

```diff
@@ -131,8 +131,8 @@ urlpatterns = [
 ```
 
 <br/>
-Done! Now you can proceed onto configuring your models to generate GraphQL types that adopt their stucture :tada:
-_Your graphql endpoint is available at http://localhost:8000/graphql/_
+Done! Now you can proceed onto configuring your models to generate GraphQL types that adopt their structure :tada:
+_Your graphql endpoint is available at http://localhost:8000/graphql_
```

Comment:
Best to leave the end `/`. Even though https://docs.djangoproject.com/en/3.0/ref/settings/#append-slash defaults to True, I've seen requests fail when querying without it because all GraphQL requests are `POST`ed

```suggestion
_Your GraphQL endpoint is available at http://localhost:8000/graphql/_
```